docs/users/interactives.rst

```diff
@@ -0,0 +1,46 @@
+=============
+Embedding Interactives
```

Comment:
if we're moving the user-facing stuff to the largo website and just keeping developer docs in the repo directly, should this be here or on the largo website? probably on the largo website?